e2e/claims.service.e2e.ts

```diff
@@ -43,14 +44,14 @@ const roleName2 = 'myrole2';
 const roleName3 = 'myrole3';
 const roleName4 = 'myrole4';
 const namespace = root;
-const version = 1;
+const version = '1';
```

Comment:
you need `int` type only in one place, why then use `parseInt()` in a couple of places instead of  `version.toString()` in one place?